src/wpt/WebCryptoAPI-test.ts

```diff
@@ -0,0 +1,405 @@
+// Copyright (c) 2017-2022 Cloudflare, Inc.
+// Licensed under the Apache 2.0 license found in the LICENSE file or at:
+//     https://opensource.org/licenses/Apache-2.0
+
+import { type TestRunnerConfig } from 'harness/harness';
+
+export default {
+  'algorithm-discards-context.https.window.js': {
+    comment: 'ReferenceError: document is not defined',
+    skipAllTests: true,
+  },
+  'crypto_key_cached_slots.https.any.js': {
+    comment: 'Investigate this',
+    expectedFailures: [
+      'CryptoKey.algorithm getter returns cached object',
+      'CryptoKey.usages getter returns cached object',
+    ],
+  },
+
+  'derive_bits_keys/cfrg_curves_bits.js': {},
+  'derive_bits_keys/cfrg_curves_bits_curve25519.https.any.js': {},
+  'derive_bits_keys/cfrg_curves_bits_curve448.https.any.js': {
+    comment:
+      'We return TypeError, but they expect InvalidAccessError/OperationError, etc.',
+    expectedFailures: [
+      'X448 key derivation checks for all-zero value result with a key of order 0',
```

Comment:
Likely good to add a comment here that not only are these failures expected, they are *intentionally* unsupported since BoringSSL does not include support for X448